bucket/regeditor.json

```
@@ -0,0 +1,39 @@
+{
+    "version": "12.0.2172",
+    "description": "Freeware for editing the Windows Registry.",
```

No need to specify license in description.

```suggestion
    "description": "Windows Registry editor.",
```

---

Thanks, the description is copy from the website without change, and it is fixed now.